runner.sh

```diff
@@ -1,11 +1,33 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
```

Comment:
nit: you don't need bash arrays for this, use regular vars and backslash to escape newlines. I'm sure there is a use case for running in a smaller Docker container without bash.

Reply:
ah that makes so much sense - i was confused as to why it didn't work! will change